CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Unreleased
 
+### Enhancements
+* Compatibility with ruby 2.7.0 and 2.6.5. [@tegon](https://github.com/tegon), [@pedrofurtado](https://github.com/pedrofurtado) and [@feliperenan](https://github.com/feliperenan)
```

Comment:
This don't need a changelog entry. It has no code change, so it means that the current version is already compatible.

Reply:
@rafaelfranca Thanks for feedback! Done 🤝